video/out/w32_common.c

```diff
@@ -1729,11 +1757,13 @@ static void w32_api_load(struct vo_w32_state *w32)
     w32->api.pAdjustWindowRectExForDpi = !user32_dll ? NULL :
                 (void *)GetProcAddress(user32_dll, "AdjustWindowRectExForDpi");
 
-    // imm32.dll must be loaded dynamically
-    // to account for machines without East Asian language support
     HMODULE imm32_dll = LoadLibraryW(L"imm32.dll");
```

Comment:
Do you know why we load it dynamically? Docs says
> Minimum supported client: Windows XP [desktop apps only],East Asian language support installed.

But I think it is "OR", no? Either way, is this available on Windows 10+? What platforms need dynamic loading?